src/lib.rs

```diff
@@ -1167,6 +1179,26 @@ impl Drop for BufferWriteMapping {
 }
 
 impl Buffer {
+    /// Use only a portion of this Buffer for a given operation. Choosing a range with 0 size will
+    /// return a slice that extends to the end of the buffer.
+    pub fn slice<S: std::ops::RangeBounds<BufferAddress>>(&self, bounds: S) -> BufferSlice {
+        let offset = match bounds.start_bound() {
+            std::ops::Bound::Included(&bound) => bound,
```

Comment:
can we have `use std::ops::{Bound, RangeBounds}` somewhere at the start of the file?